src/llm/llama.ipynb

```diff
@@ -0,0 +1,153 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Credits: https://medium.com/@murtuza753/using-llama-2-0-faiss-and-langchain-for-question-answering-on-your-own-data-682241488476"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
What is going to be the purpose of this notebook? Just a sample notebook that is a cookbook on how to work with llama?

Reply:
It no longer exists. Invalid question from now on.